src/contexts/config/Config.ts

```diff
@@ -34,13 +34,21 @@ export type Config = {
          */
         siteCode: string
         /**
-         * quotaId is the "quota id" we pass to the filter quota cost
+         * kasQuotaId is the "quota id" we pass to the filter quota cost for kafka
          */
-        quotaId: string,
+        kasQuotaId: string,
         /**
-         * trialQuotaId is the "quota id" we pass to the filter quota cost for trial instances
+         * kasTrialQuotaId is the "quota id" we pass to the filter quota cost for trial instances for kafka
          */
-        trialQuotaId: string
+        kasTrialQuotaId: string
```

Comment:
This is going to explode, so it's probably worth introducing some nested objects here. We also have specific config for specific services, which might be a better place to put it.